src/LicenseConfiguration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lendable\ComposerLicenseChecker;
+
+final class LicenseConfiguration
+{
+    /**
+     * @param list<string> $allowedLicenses
+     * @param list<string> $allowedPackagePatterns
+     */
+    public function __construct(public readonly array $allowedLicenses, public readonly array $allowedPackagePatterns)
```

Comment:
Feature request: a way to add licenses/package patterns to the existing LicenseConfiguration.

Use-case: this library provides a base that should be working OOTB for most of our products, but the existing ones might need to customise it a bit to allow for some exceptions.

Reply:
I was thinking you would include sets via:

```php
$config->allowLicenses(...LicenseSetList::companyWideAccepted())
```

Reply:
Note that `allowLicenses()` appends so you can call it multiple times if you wanted to package the ability to expose a builder in a partial state to then extend from.

Reply:
The perfect solution for me would allow for copying all of the defined configurations (licenses, packages) so that it's impossible to omit.

Eg. `LicenseConfiguration::merge(new SharedLicenseConfiguration(), new LicenseConfiguration(...))` returning the merged configuration.